DependencyInjection/Compiler/AnnotationConfigurationPass.php

```diff
@@ -109,7 +109,7 @@ private function getClassName($filename)
     {
         $src = file_get_contents($filename);
 
-        if (!preg_match('/\bnamespace\s+([^;]+);/s', $src, $match)) {
+        if (!preg_match('/\bnamespace\s+([^;\{\s]+)\s*?[;\{]/s', $src, $match)) {
```

Comment:
the ``?`` here is about making the ``*`` quantifier non-greedy. It is *not* the same ``?`` than the one making a character optional: http://php.net/manual/fr/regexp.reference.repetition.php